### PR TITLE
docs: expire after

### PR DESCRIPTION
**Pull request by discord9**
## What's Changed in this PR

*Describe the change in this PR*

uncomment the `expire after` part, and update the `create flow` syntax

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/nightly/zh/user-guide/continuous-aggregation/manage-flow.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ AS
 它可以是一个现有表或一个新表，`flow` 会在目标表不存在时创建它。
 但如果表已经存在，表的 schema 必须与查询结果的模式匹配。
 
-<!-- `expire after` 是一个可选的时间间隔，用于从源表中过期数据。过期时间是相对于当前时间的时间（“当前时间”是指数据到达 Flow 引擎的物理时间）。例如，`INTERVAL '1 hour'` 表示数据 **早于** 1 小时的数据将被过期。过期数据将直接被删除。 -->
+`expire after` 是一个可选的时间间隔，用于从源表中过期数据。过期时间是相对于当前时间的时间（“当前时间”是指数据到达 Flow 引擎的物理时间）。例如，`INTERVAL '1 hour'` 表示数据 **早于** 1 小时的数据将被过期。过期数据将直接被删除。请注意，“删除”意味着数据将从 flow 的内部状态中删除，而不是源表，因此您的数据是安全的，只是 flow 将拒绝计算到达时间太晚的数据。
```
Comment (tisonkun) [inline docs/nightly/zh/user-guide/continuous-aggregation/manage-flow.md:43]:
```suggestion
`expire after` 是一个可选的时间间隔，用于从源表中过期数据。过期时间是相对于当前时间的时间（“当前时间”是指数据到达 Flow 引擎的物理时间）。例如，`INTERVAL '1 hour'` 表示数据**早于** 1 小时的数据将被过期。过期数据将直接被删除。请注意，“删除”意味着数据将从 flow 的内部状态中删除，而不是源表，因此您的数据是安全的，只是 flow 将拒绝计算到达时间太晚的数据。
```

File: docs/nightly/en/user-guide/continuous-aggregation/manage-flow.md [hunk 2]
```diff
@@ -38,32 +38,22 @@ AS
...
 
-<!-- `expire after` is an optional interval to expire the data from the source table. The expiration time is a relative time from the current time (by "current time" we means the physical time of the data arrive the Flow engine). For example, `INTERVAL '1 hour'` means the data **older** than 1 hour from the current time will be expired. Expired data will be dropped directly. -->
+`expire after` is an optional interval to expire the data from the source table. The expiration time is a relative time from the current time (by "current time" we means the physical time of the data arrive the Flow engine). For example, `INTERVAL '1 hour'` means the data **older** than 1 hour from the current time will be expired. Expired data will be dropped directly. Notice that "dropped" means the data will be removed from the flow's internal state, not the source table, so you data is safe, just flow will refuse to compute data that arrive too late.
```
Comment (tisonkun) [inline docs/nightly/en/user-guide/continuous-aggregation/manage-flow.md:41]: IIRC in docs it's normally not use "we"?

**Review by nicecui — APPROVED** [review]

File: docs/nightly/en/user-guide/continuous-aggregation/manage-flow.md [hunk 3]
```diff
@@ -36,34 +28,47 @@ AS
...
+EXPIRE AFTER INTERVAL '1 hour'
 COMMENT = "My first flow in GreptimeDB"
 AS
 SELECT count(item) from my_source_table GROUP BY tumble(time_index, INTERVAL '5 minutes', '2024-05-20 00:00:00');
 ```
 
-The created flow will compute `count(item)` for every 5 minutes and store the result in `my_sink_table`. For the `tumble()` function, refer to [define time window](./define-time-window.md) part.
+The created flow will compute `count(item)` for every 5 minutes and store the result in `my_sink_table`. All data comes within 1 hour will be used in the flow. For the `tumble()` function, refer to [define time window](./define-time-window.md) part. 
+
+### `EXPIRE AFTER` clause
```
Comment (nicecui) [inline docs/nightly/en/user-guide/continuous-aggregation/manage-flow.md:56]: @waynexia PTAL

File: docs/nightly/zh/user-guide/continuous-aggregation/manage-flow.md [hunk 4]
```diff
@@ -59,16 +53,30 @@ SELECT count(item) from my_source_table GROUP BY tumble(time_index, INTERVAL '5
...
 COMMENT = "My first flow in GreptimeDB"
 AS
 SELECT count(item) from my_source_table GROUP BY tumble(time_index, INTERVAL '5 minutes', '2024-05-20 00:00:00');
 ```
 
-该 Flow 将每 5 分钟计算 `count(item)` 并将结果存储在 `my_sink_table` 中。
-有关 `tumble()` 函数，请参考[定义时间窗口](./define-time-window.md) 部分。
+创建的 flow 将每 5 分钟计算 `count(item)` 并将结果存储在 `my_sink_table` 中。所有在 1 小时内的数据将在 flow 中使用。有关 `tumble()` 函数，请参考[定义时间窗口](./define-time-window.md) 部分。
+
+### `EXPIRE AFTER` 语句
```
Comment (nicecui) [inline docs/nightly/zh/user-guide/continuous-aggregation/manage-flow.md:64]: @waynexia PTAL

File: docs/nightly/en/user-guide/continuous-aggregation/manage-flow.md [hunk 5]
```diff
@@ -36,34 +28,47 @@ AS
...
+The data timestamp is the time stored in the time index column of the source table,
+while the processing time refers to the moment when the Flow engine executes the aggregation operation.
+
+The `EXPIRE AFTER` clause specifies the interval after which the data will expire.
+Any data with a timestamp older than the processing time minus the interval time will be expired.
+This means that the Flow engine only uses data within this interval to compute the aggregation.
+
+For example, if the Flow engine processes the aggregation operation at 10:00:00 and the `INTERVAL '1 hour'` is set,
+any data older than 1 hour from the processing time (data before 09:00:00) will be expired.
+Only data within the interval from 09:00:00 to 10:00:00 will be used in the aggregation.
```
Comment (waynexia) [inline docs/nightly/en/user-guide/continuous-aggregation/manage-flow.md:69]: All the data after 09:00:00 will be used. Like a future time 11:00:00

**Review by waynexia — APPROVED** [review]
others LGTM

**Pull request merged**